config/locales/models/assignments/en.yml

```diff
@@ -25,7 +25,7 @@ en:
         remark_due_date: Remark Due Date
         remark_message: Remark Request Instructions
         repository_folder: Repository Folder
-        section_due_dates_type: Enable section due dates
+        section_due_dates_type: Enable section properties
```

Comment:
I'm thinking about what wording makes the most sense to a typical user here. What do you think about:

- Enable section specific settings
- Set section specific settings
- Set settings for each section
- ... 

Something like that? I don't know if calling it section properties makes the most sense to a user (who doesn't have a view of the database and knows that's what the table name/ model name is.

Reply:
Agreed, "section properties" doesn't make a ton of sense for end users. I like "Enable section specific settings", because it seems to me to be the most precise description of what checking the box does. If I saw "set", I would probably assume that it will automatically generate some sort of default. It kind of does this for the hidden property, but doesn't (at least not visibly) for due date. If checking the section-specific settings box prefilled the section's due date with the assignment's due date, for example, I'd feel a bit better about using set. Either way, I've gone with section specific preferences, but none of these are strong preferences.

Reply:
That makes sense, why don't we go with "Enable section specific settings" then